mkdocs-website/docs/en/API/events.md

```
@@ -0,0 +1,130 @@
+
+`waills.Events.Off(name: string)`
+:   Removes an event listener with the specificed name
+
+
+## Event Types
+
+### ApplicationEvent
+Returned when an application hook event is triggered. The event can be cancelled by calling the `Cancel()` method on the event.
+```go
+type ApplicationEvent struct {
+	Id        uint
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)